packages/astro/test/0-css.test.js

```diff
@@ -43,7 +44,7 @@ describe('CSS', function () {
 				expect(el2.attr('class')).to.equal(`visible ${scopedClass}`);
 
 				// 2. check CSS
-				const expected = `.blue.${scopedClass}{color:#b0e0e6}.color\\\\:blue.${scopedClass}{color:#b0e0e6}.visible.${scopedClass}{display:block}`;
+				const expected = `.blue.${scopedClass}{color:powderblue}.color\\\\:blue.${scopedClass}{color:powderblue}.visible.${scopedClass}{display:block}`;
```

Comment:
The only reason this test change concerns me is because this might suggest preprocessing is not working the same way? I could have sworn that these optimizations were happening during dev with the `preprocessStyle` hook in `vite-plugin-astro`. Could be wrong, I'll double check this locally.

Reply:
Before this was changed because we ran CSS through esbuild, see here: https://github.com/withastro/astro/pull/3439/files#diff-6b416e91b57562a024efa9947ff14a340bdd0edbca71f3c3d08531d9dfa29ea3L80 which we are not doing anymore (deferring to vite:css-post which is perhaps only doing that if `minify: true` is enabled.

Reply:
Should we default to `minify: true` to maintain the same behavior, in that case?

Reply:
Added that in https://github.com/withastro/astro/pull/3439/commits/4fdefbddf43926d77dc72d50447fd4c58b032744